lib/cmake-scripts/cntp-extrainstalldirs.cmake

```diff
@@ -1,11 +1,20 @@
 include_guard()
 
 function(cntp_define_build_dirs)
-
+    set(QMAKE_EXTRA_ARGS)
     IF(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-        find_path(QMAKE_PATH qmake6.exe
+        find_path(QMAKE_PATH
+                NAMES qmake6.exe
```

Comment:
`find_program` instead? To clean up the `if (WINDOWS)` below

Reply:
Maybe we can do this as part of another PR